registration-processor/core-processor/registration-processor-uin-generator-stage/src/test/java/io/mosip/registration/processor/stages/uigenerator/UinGeneratorStageTest.java

```diff
@@ -229,6 +229,9 @@ public void consumeAndSend(MosipEventBus mosipEventBus, MessageBusAddress fromAd
 	@Mock
 	private IdSchemaUtil idSchemaUtil;
 
+	@Value("${mosip.regproc.max.retrycount}")
+	Integer maxRetrycount=3;
```

Comment:
No need to set here as your setting with ReflectionTestUtils.setField in setup method and set to specific value in respective test case

Reply:
Removed.